pkg/helm/build.go

```diff
@@ -9,7 +9,9 @@ import (
 )
 
 // These values may be referenced elsewhere (init.go), hence consts
-const helmClientVersion string = "v2.15.2"
+var helmClientVersion string
```

Comment:
I'd favor removing the need for this global var and instead use the helper `getHelmVersion` when needed (as is used below).  (So, in `init_test.go`, etc., we can switch out `helmClientVersion` with `helmDefaultClientVersion` or `getHelmVersion(...)`.)  Would this be possible?

Reply:
Sure, will get this done by the end of the day

Reply:
Hello @vdice, Actually  `helmClientVersion` is used in (init/initTest) to determine if the client is at the same version with the server otherwise it won't init `Tiller`, also used in (build/buildTest). So we want to know at any time which is the currently active version.  So  This global is kind of required unless we attach a  Version property the Mixin context and provide a getter/setter for further use?

Reply:
I see.  Yes, let's try the latter approach of adding `Version`(or `HelmClientVersion`) to the `Mixin` struct in `helm.go` and then setting it appropriately here.  Then, in `init.go`, we should be able to access via `m.Version` (or `m.HelmClientVersion`).

Reply:
You will notice `const defaultHelmClientVersion string = "v2.15.2"`has been moved to `helm.go`. do you think it won't confuse people since it was pretty much tied to `build.go`?

Reply:
That's definitely a valid concern... I think I'm still in favor of the current direction with the benefits it brings...